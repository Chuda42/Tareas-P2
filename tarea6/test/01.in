# Prueba de operaciones de mapping
estaLlenoMap
existeAsociacionEnMap 5
asociarEnMap 5 1.5
existeAsociacionEnMap 5
estaLlenoMap
desasociarEnMap 5
existeAsociacionEnMap 5
estaLlenoMap
asociarEnMap 5 2.5
existeAsociacionEnMap 5
valorEnMap 5
asociarEnMap 10 1.2
asociarEnMap 20 1.3
asociarEnMap 90 1.4
asociarEnMap 80 1.5
asociarEnMap 70 1.6
asociarEnMap 60 1.7
asociarEnMap 40 1.8
asociarEnMap 30 1.9
valorEnMap 5
valorEnMap 30
estaLlenoMap
asociarEnMap 100 1.0
estaLlenoMap
Fin
