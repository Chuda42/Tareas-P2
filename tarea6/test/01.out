1>#  Prueba de operaciones de mapping.
2>No lleno.
3>No existe asociacion.
4>Establecida la asociación.
5>Existe asociacion.
6>No lleno.
7>Eliminada la asociación.
8>No existe asociacion.
9>No lleno.
10>Establecida la asociación.
11>Existe asociacion.
12>2.50
13>Establecida la asociación.
14>Establecida la asociación.
15>Establecida la asociación.
16>Establecida la asociación.
17>Establecida la asociación.
18>Establecida la asociación.
19>Establecida la asociación.
20>Establecida la asociación.
21>2.50
22>1.90
23>No lleno.
24>Establecida la asociación.
25>LLeno.
26>Fin.
